microsite/blog/2025-05-12-from-zero-to-maintainer-my-opensource-journey-with-backstage.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: 'From Zero to Maintainer: My Open Source Journey with Backstage'
+author: Kashish Mittal, Red Hat
+authorURL: https://github.com/04kash
```

Comment:
Can we also add: `authorImageURL: https://avatars.githubusercontent.com/u/113269381?v=4`, that would pull your GitHub profile image